third-party/crowdsec/pom.xml

```diff
@@ -49,6 +55,22 @@
             <artifactId>jackson-datatype-jdk8</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jdk8</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java-transport-zerodep</artifactId>
+            <version>3.3.0</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java-transport-zerodep</artifactId>
+            <version>3.3.0</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
Duplicated. Why do we need this dependency BTW?

Reply:
Because I only use the prompt of IDE :(, I will recheck pom.xml before pushing

Reply:
The question was as well what do we need this dependency for? :)

Reply:
Maybe due to my old code, I fixed it up already